content/courses/program-security/owner-checks.md

```diff
@@ -3,10 +3,10 @@ title: Owner Checks
 objectives:
   - Explain the security risks associated with not performing appropriate owner
     checks
-  - Implement owner checks using long-form Rust
-  - Use Anchor’s `Account<'info, T>` wrapper and an account type to automate
+  - Implement owner checks using native Rust
+  - Use Anchor's `Account<'info, T>` wrapper and an account type to automate
```

Comment:
Just swap these two (and do it everywhere, Anchor is default for onchain programs)